api/auth.go

```diff
@@ -13,13 +13,17 @@ import (
 
 // requireAuthentication checks incoming requests for tokens presented using the Authorization header
 func (a *API) requireAuthentication(w http.ResponseWriter, r *http.Request) (context.Context, error) {
-	token, err := a.extractBearerToken(w, r)
-	config := getConfig(r.Context())
-	if err != nil {
-		a.clearCookieTokens(config, w)
-		return nil, err
+	var token string
+	config := a.getConfig(r.Context())
+	if cookie, err := r.Cookie(config.Cookie.Key + "-access-token"); err == nil {
```

Comment:
May have missed/forgot the discussion -- what's the reason for checking the cookie access token before checking the authorization headers again?

Reply:
iirc the plan was to prefer storing the access and refresh tokens in the cookie over the header